src/main/webapp/resources/js/components/samples/components/GenomeAssemblyList.jsx

```diff
@@ -30,20 +35,86 @@ export function GenomeAssemblyList({ removeSampleFiles = () => {} }) {
     downloadGenomeAssemblyFile({ sampleId, genomeAssemblyId });
   };
 
+  /*
+  Set default genome assembly for sample to be used for analyses
+   */
+  const updateDefaultGenomeAssembly = (genomeAssembly) => {
+    const { fileInfo: genomeAssemblyObj } = genomeAssembly;
+
+    updateSampleDefaultGenomeAssembly({
+      sampleId: sample.identifier,
+      genomeAssemblyId: genomeAssemblyObj.identifier,
+    })
+      .then(({ data }) => {
+        dispatch(setDefaultGenomeAssembly(genomeAssemblyObj));
+        notification.success({ message: data.message });
+      })
+      .catch((error) => {
+        notification.error({ message: error });
+      });
+  };
+
   /*
    Get the actions required for a Genome Assembly
    */
-  const getActionsForGenomeAssembly = (genomeAssembly) => {
-    let actions = [
+  const getActionsForGenomeAssembly = (genomeAssembly, index) => {
+    const { fileInfo: genomeAssemblyObj } = genomeAssembly;
+    let actions = [];
+
+    if (isModifiable) {
+      if (
+        (sample.defaultGenomeAssembly !== null &&
+          genomeAssemblyObj.identifier ===
+            sample.defaultGenomeAssembly.identifier) ||
+        (sample.defaultGenomeAssembly === null && index === 0)
+      ) {
+        actions.push(
+          <Tooltip
+            title={i18n("SampleFilesList.defaultSelectedAssembly")}
+            placement="top"
+            key={`default-tag-tooltip-ga-${genomeAssemblyObj.identifier}`}
+          >
+            <Tag
+              color="#108ee9"
```

Comment:
What is this color chosen from?

Reply:
Got this color from the ant design tag component page.It's in one of the examples.

Reply:
I added in css vars for blue based on one of your branches and updated the tag here and in sequenceobjectlist.jsx to use them in aabc9d4a566406d9ef2235054febf0b8a29d4d7f